### PR TITLE
Tca 37 implement test for login error

### PR DESCRIPTION
**Pull request by pscacchi**
The  error path login test was implemented and others tests was updated.
The test of Constants value will be implemented in the next PR

### CODE REVIEW

File: app/src/test/java/ar/scacchipa/twittercloneapp/repository/AuthorizationRepositoryTest.kt [hunk 1]
```diff
@@ -46,6 +48,47 @@ class AuthorizationRepositoryTest {
 
         Assert.assertEquals(expected, actual)
     }
+
+    @Test
+    fun whenAuthRepoReceiveRequestWithoutBodyReturnError() = runTest {
```
Comment (sebastianMullerEndava) [inline app/src/test/java/ar/scacchipa/twittercloneapp/repository/AuthorizationRepositoryTest.kt:53]: Seems ok but here we use mockito to model de mock behavior , which is ok, but in the next test we create the mock by hand, maybe we could go for mockito in all the test.

File: app/src/test/java/ar/scacchipa/twittercloneapp/viewmodel/AuthWebDialogViewModelTest.kt [hunk 2]
```diff
@@ -81,6 +71,15 @@ class AuthWebDialogViewModelTest {
         Assert.assertTrue(
             subject.tokenResource.value == TokenResource.Cancel)
     }
+    @Test
+    fun webViewShouldWebResourceErrorToSubject() {
```
Comment (sebastianMullerEndava) [inline app/src/test/java/ar/scacchipa/twittercloneapp/viewmodel/AuthWebDialogViewModelTest.kt:75]: something is missing in the name of the test to be clear, maybe a verb after should

**Review by sebastianMullerEndava — APPROVED** [review]

**Review by Diana-Endava — APPROVED** [review]
good job!, let's keep working with the naming maybe not use "should/should not" and directly use something like "generateSuccessToken/generateEmptyToken" etc

File: app/src/test/java/ar/scacchipa/twittercloneapp/viewmodel/AuthWebDialogViewModelTest.kt (hunk 2 above)
Comment (pscacchi) [reply app/src/test/java/ar/scacchipa/twittercloneapp/viewmodel/AuthWebDialogViewModelTest.kt:75]: Now, the function is called webViewSendWebResourceErrorToSubject()

**Pull request merged**